stage6/03-gr-iio/00-run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+on_chroot << EOF
+
+git clone https://github.com/analogdevicesinc/gr-iio.git
+pushd gr-iio
+cmake .
+make
```

Comment:
can we parametrize the number of jobs in the `make` command[s]?
something like:
```
NUM_JOBS=${NUM_JOBS:-1}
make -j$NUM_JOBS`
```
that way we can alloc more threads/jobs for the build and save some time

Reply:
i am not sure how all of these will propagate from the build script to these individual ones. If i add it to the build.sh will it be visible in these secondary ones when they get called ?